### PR TITLE
finished with coding challenge

### PR DESCRIPTION
**Pull request by devonhackley**

### CODE REVIEW

**Comment by devonhackley** [comment]
- [ ] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

File: code-challenges/401challenges/src/main/java/challenges/linkedlist/LinkedList.java [hunk 1]
```diff
@@ -147,4 +147,32 @@ public void insertAfter(int afterValue, int data){
...
+        } else {
+           Node current = this.head;
+            int lengthOfList = 0;
+
+            while(current != null){
+                current = current.next;
+                lengthOfList++; // grab length of the list
+            }
+            if(lengthOfList < num){
+                throw new IllegalArgumentException("The number passed in was greater than the list"); // num was greater than length of the list
```
Comment (evanslaton) [inline code-challenges/401challenges/src/main/java/challenges/linkedlist/LinkedList.java]: I'm not sure throwing an error here is the best idea. The user doesn't know how long the LinkedList is and stopping the code if they pass in a value higher than the length of the list seems like a very serious penalty

File: code-challenges/401challenges/src/test/java/challenges/LinkedListTest.java [hunk 2]
```diff
@@ -3,6 +3,7 @@
 import challenges.linkedlist.LinkedList;
 
 import org.junit.Test;
+import sun.awt.image.ImageWatched;
```
Comment (evanslaton) [inline code-challenges/401challenges/src/test/java/challenges/LinkedListTest.java]: This line is preventing me from running your tests!
